test/Interop/Cxx/class/memory-layout-silgen.swift

```diff
@@ -1,5 +1,7 @@
 // RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -emit-ir -o - %s | %FileCheck %s
 
+// UNSUPPORTED: OS=linux-android, OS=linux-androideabi
```

Comment:
The only test I disabled, because of cyclic dependency errors related to header files pulled in by the `MemoryLayout` module, in particular the `__config` C++ header. This test used to work before when no `-sdk` was passed in, because it just used to pick up `stddef.h` and other headers from the clang resource directory instead, but now it picks them up from the Android NDK sysroot first.

I've seen these errors on a bunch of the C++ Interop tests running natively on Android since April, but since this is the only one that fails when cross-compiling, simply disabled this for now.

I'm not that familiar with these header "dependency hell" issues, so if someone has a way to make it work, we could try that instead.

Reply:
Can you open a issue in bugs.swift.org and provide the link here in a comment?

Reply:
Sure, will do that later today.